modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -240,13 +240,10 @@ private void registerMustacheLambdas() {
         }
 
         if (additionalProperties.containsKey("lambda")) {
-            LOGGER.warn("A property named 'lambda' already exists. Mustache lambdas renamed from 'lambda' to '_lambda'. " +
-                    "You'll likely need to use a custom template, " +
```

Comment:
Has the detailed message moved elsewhere? If not, we should keep the link to templating docs.

Reply:
Hi @jimschubert 

I do not understand the question. 
Condition `if (additionalProperties.containsKey("lambda"))` should never evaluate to true and if then only because something has changed in DefautlCodegen and that should be caught by a first build.

I think the construct bellow made sense when it was in a generator derived from DefaultCodegen, but not now. 
```
 if (additionalProperties.containsKey("lambda")) {
            LOGGER.warn("A property named 'lambda' already exists. Mustache lambdas renamed from 'lambda' to '_lambda'. " +
                    "You'll likely need to use a custom template, " +
                    "see https://github.com/OpenAPITools/openapi-generator/blob/master/docs/templating.md. ");
            additionalProperties.put("_lambda", lambdas);
        } else {
            additionalProperties.put("lambda", lambdas);
        }
``` 

Makes sense?

Reply:
I see, thanks for the clarification. I incorrectly remembered your earlier change being in DefaultGenerator, and missed that it's actually called in this class constructor.